cmd/apiserver-boot/boot/glide.go

```diff
@@ -63,39 +63,31 @@ func fetchGlide() {
 }
 
 func copyGlide() {
-	// copy the files
+	// Move up two directories from the location of the `apiserver-boot`
+	// executable to find the `vendor` directory we package with our
+	// releases. TODO(campbellalex@google.com): this doesn't work for people
```

Comment:
Ideally the src would be a tar.gz nested in the original tar.gz, so this would just be untaring instead of copying